### ISSUE TITLE
Unable to import from @influxdata/influxdb-client in an ES module

### ISSUE THREAD

**Issue by q42jaap** [influxdata/influxdb-client-js#459]
I'm creating a package using typescript 4.7 with `"type": "module"` in package.json (to create a es2021 module instead of commonjs). 

__Expected Behavior:__
I should be able to import code from `@influxdata/influxdb-client` in a typescript environment.

__Current Behavior:__
The typescript compiler refuses to import code from `@influxdata/influxdb-client`:

```
src/influx.ts:1:22 - error TS7016: Could not find a declaration file for module '@influxdata/influxdb-client'. '/Users/jaap/Projects/LY/tmp2/node_modules/@influxdata/influxdb-client/dist/index.mjs' implicitly has an 'any' type.
  Try `npm i --save-dev @types/influxdata__influxdb-client` if it exists or add a new declaration (.d.ts) file containing `declare module '@influxdata/influxdb-client';`

1 import { flux } from '@influxdata/influxdb-client';
```

__Possible Solution:__

When removing the `"exports": { ... }` section from `@influxdata/influxdb-client/package.json` the code will import correctly.

__Steps/Code to Reproduce the Problem:__

1. Checkout https://github.com/q42jaap/influxdata-influxdb-client-issue
1. run `npm install`
2. run `npm run build`

__Specifications:__

- Version: @influxdata/influxdb-client 1.27.0
- Platform: node 16.15, typescript 4.7.4

**Comment by sranka** [comment]
Thank you @q42jaap for using the client and reporting an issue with a perfect description.

**Comment by sranka** [comment]
This issue is reproducible only with typescript 4.7 where types must be defined in `exports` section of `package.json` as described at the end of https://www.typescriptlang.org/docs/handbook/esm-node.html .

**Comment by sranka** [comment]
@q42jaap Thank you again for the git repository that makes this super-easy to reproduce. The fix is already available in the nightly build, it works after changing `package.json` dependencies to:

```js
  "dependencies": {
    "@influxdata/influxdb-client": "1.28.0-nightly.2082"
  },
```
The fix will be part of 1.28 release (approx mid-summer).

**Comment by q42jaap** [comment]
@sranka Thanks for your nice words, I would like to remark that this repository is well-maintained and it's awesome to see this is picked up so fast!
